content/riak/kv/3.0.13/configuring/next-gen-replication.md

```diff
@@ -23,7 +23,11 @@ aliases:
 The configuration for Next Gen Replication is kept in
  the `riak.conf` configuration file.
 
-## Settings
+{{% note %}}
+Next Gen Replication relies on the TicTac AAE system, which needs to be enabled and configured. See the [TicTac AAE configuration][configure tictacaae] documentaion.
```

Comment:
Same thing about link targets and notes again. This will be true for all versions so I won't mention it again.